core/tauri/src/menu/builders/mod.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019-2023 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+#![cfg(desktop)]
+
+//! A module containting menu builder types
+
+pub use muda::builders::AboutMetadataBuilder;
```

Comment:
we should create a wrapper type here instead (both for the builder and the metadata struct), muda is in alpha so we can't re-export it (would also be harder to track breaking changes).

Reply:
We are already re-exporting a bunch of types from `muda` and `tray-icon` and as long as we specify `0.x` version in `Cargo.toml`, we should be fine no?

I seriously doubt that these types will change in muda after this PR is merged but we could give it a test run for a while through the alpha and beta.